### ISSUE TITLE
[Bug]: A11y addon with `manual: true` gets stuck in loading state

### ISSUE THREAD

**Issue by edoardocavazza** [storybookjs/storybook#30314]
### Describe the bug

When the A11y addon is configured with `manual: true` it does not work anymore.
The UI got stuck in the loading state with the message:
> Please wait while the accessibility scan is running

### Reproduction link

https://stackblitz.com/edit/github-h1gyzzpc-ab5abiso?file=.storybook%2Fpreview.ts&preset=node

### Reproduction steps

* Go to the Button Primary story
* Switch to the Accessibility tab

### System

```bash
Storybook Environment Info:

  System:
    OS: Linux 5.0 undefined
    CPU: (8) x64 Intel(R) Core(TM) i9-9880H CPU @ 2.30GHz
    Shell: 1.0 - /bin/jsh
  Binaries:
    Node: 18.20.3 - /usr/local/bin/node
    Yarn: 1.22.19 - /usr/local/bin/yarn <----- active
    npm: 10.2.3 - /usr/local/bin/npm
    pnpm: 8.15.6 - /usr/local/bin/pnpm
  npmPackages:
    @storybook/addon-a11y: ^8.5.0 => 8.5.0 
    @storybook/addon-essentials: ^8.5.0 => 8.5.0 
    @storybook/blocks: ^8.5.0 => 8.5.0 
    @storybook/test: ^8.5.0 => 8.5.0 
    @storybook/web-components: ^8.5.0 => 8.5.0 
    @storybook/web-components-vite: ^8.5.0 => 8.5.0 
    storybook: ^8.5.0 => 8.5.0
```

**Comment by edoardocavazza** [comment]
It seems that Storybook now expects `a11y: { manual: true }` to be part of the `globals` setting.
I changed my configuration from:
```ts
const preview: Preview = {
  parameters: {
    a11y: {
      manual: true,
    },
  },
};
```
to:
```ts
const preview: Preview = {
  parameters: {
    globals: {
      a11y: {
        manual: true,
      },
    },
  },
};
```
and it works fine now.

If this is an intentional configuration change, we can close the issue.

**Comment by shilman** [comment]
@edoardocavazza I think it's still a bug. We shouldn't be making breaking changes in minor releases. But at least you have a workaround now! 😁

**Comment by shilman** [comment]
@edoardocavazza As a side note, can you talk a little bit about your use case for the `manual` flag? As we were updating the addon we were curious whether people were using it and how.

**Comment by valentinpalkovic** [comment]
Not relevant anymore, since `parameters.a11y.manual` does not exist anymore in SB 9.0
